HelpSource/Classes/ArrayedCollection.schelp

```diff
@@ -247,6 +247,32 @@ y.postln;
 )
 ::
 
+method::boundedInsert
+Same as link::#-insert::, but it maintains the size of the receiver, and the values for index greater than the size of the link::Classes/ArrayedCollection:: will be clipped to the last index.
+
+code::
+~arrayA = [1, 2, 3]             // -> [1, 2, 3]
+~arrayA.maxSize                 // -> 4
+
+~arrayA.boundedInsert(0, \a)    // -> [a, 1, 2]
+~arrayA                         // -> [a, 1, 2]
+~arrayA.boundedInsert(-2, '-a') // -> [-a, a, 1]
+~arrayA                         // -> [-a, a, 1]
+~arrayA.boundedInsert(20, \a)   // -> [-a, a, a]
+~arrayA                         // -> [-a, a, a]
+
+~arrayA = [1, 2, 3]                       // -> [1, 2, 3]
+~arrayB = ~arrayA.boundedInsert(0, \a)    // -> [a, 1, 2]
+~arrayA                                   // -> [a, 1, 2]
+~arrayB                                   // -> [a, 1, 2]
+~arrayC = ~arrayA.boundedInsert(20, \a)   // -> [a, 1, a]
+~arrayA                                   // -> [a, 1, a]
+~arrayC                                   // -> [a, 1, a]
+~arrayD = ~arrayA.boundedInsert(-2, '-1') // -> [-1, a, 1]
+~arrayA                                   // -> [-1, a, 1]
+~arrayD                                   // -> [-1, a, 1]
+::
+
```

Comment:
Could the examples be shorter and focus on a few core points we want to communicate about this method? It could be useful to take the help for Array#-insert as an inspiration

Reply:
Um... there as a problem:
```
(
// in this case a new object is returned
var y, z;
z = [1, 2, 3, 4];
y = z.boundedInsert(1, 999);
z.postln;
y.postln;
)
```
returns
```
[1, 999, 2, 3]
[1, 999, 2, 3]
-> [1, 999, 2, 3]
```
The problem seems to be the inconsistent behaviour of the array:
```
a = [1, 2, 3, 4] // -> [1, 2, 3, 4]
b = a.insert(1, 99) // -> [1, 99, 2, 3, 4]
a                   // -> [1, 2, 3, 4]
b                   // -> [1, 99, 2, 3, 4]

a = [1, 2, 3] // -> [1, 2, 3]
b = a.insert(1, 99) // -> [1, 99, 2, 3]
a                   // -> [1, 99, 2, 3]
b                   // -> [1, 99, 2, 3]
```

Reply:
See https://github.com/supercollider/supercollider/pull/6394/commits/5e18589a240aa319f02e7744454cb8b2daf754e3.